opat/drupal-9.yaml

```diff
@@ -746,5 +746,301 @@ chapters:
           - name: "web"
             adherence:
               level: "not-applicable"
+  # Note that this will likely be replaced with an automated process.
+  functional_performance_criteria:
+    - num: "302.1"
+      name: Without Vision
+      level: "supports"
+      notes: Testing has been done with JAWS, NVDA and VoiceOver.
+    - num: "302.2"
+      name: With Limited Vision
+      level: "supports"
+      notes: Testing has been done with browser zoom and ZoomText.
+    - num: "302.3"
+      name: Without Perception of Color
+      level: "supports"
+      notes: The interface has been reviewed for use of color.
+    - num: "302.4"
+      name: Without Hearing
+      level: "not-applicable"
+    - num: "302.5"
+      name: With Limited Hearing
+      level: "not-applicable"
+    - num: "302.6"
+      name: Without Speech
+      level: "not-applicable"
+    - num: "302.7"
+      name: With Limited Manipulation
+      level: "supports"
+      notes: Drupal's interface does not restrict users with limited manipulation.
+    - num: "302.8"
+      name: With Limited Reach and Strength
+      level: "supports"
+      notes: Drupal's interface does not restrict users with limited reach or strength.
+    - num: "302.9"
+      name: With Limited Language, Cognitive, and Learning Abilities
+      level: "not-applicable"
+  # Chapter 4
+  # This is included as an exmple. It should not need to be included for exclusively web based ACRs.
   hardware:
     notes: "Drupal is a web application. Hardware accessibility criteria is not applicable."
+    - num: "402.2.1"
```

Comment:
So @dmundra keep the num as the common reference & move "name: Information Displayed On-Screen" to the catalog.  So the drupal-9.yaml file would just have:
```
    - num: "402.2.1"
      level: "not-applicable"
      notes: If there are any that we want to add here.  
```

I can remove the name from this file, but want to confirm first. You can grab the num & name combination from my earlier comment where I started from.

Reply:
Yes exactly with the structure including the components adherence, so it would look like the following in `drupal-9.yaml`

```
      - num: "402.2.1"
        components:
          - name: "none"
            adherence:
              level: "not-applicable"
              notes: If there are any that we want to add here.
```

I used "none" so I could keep the structure for hardware and other chapters consistent with A/AA where we use "web", "authoring-tool" and so on.

Reply:
Ok, I've adopted this here. Look ok?

Reply:
Looking good for this change.